contents/blog/customer-churn-analysis-guide.md

```diff
@@ -0,0 +1,296 @@
+
+Trust me, it's less complicated than it looks.
+
+![new user retention](../images/blog/retention-vs-churn/retention-table-example-all.png)
+
+Above is a PostHog retention table for an imaginary Dropbox clone called Hedgebox. In this retention table we're looking at weekly repeat usage,  configured as:
+
+> `Unique users` who `signed up for the first time` in the last 8 weeks who then came back and triggered the `interacted with file` event in PostHog.
+
+To read it, just start on the left and work your way across:
+
+- **Cohort** Denotes the week these users signed up – i.e. the cohort these users belong to.
+
+- **Size:** Shows the size of that cohort – a useful reference to avoid comparing a very small cohort to a much larger one.
+
+- **Week 0:** Gives you the percentage of users in the cohort who used Hedgebox in that week. As it's week 0, this number will always be 100%.
+
+- **Week 1-8:** Shows the percentage of users in that cohort who returned to the product in each subsequent week – white boxes indicate periods that 'still in progress', blue boxes are completed periods.
+
+In this example, we can see retention settles at ~20% from 'week 4' onwards in most cohorts. 
+
+That's a useful baseline, but what happens if we filter by just people who use Microsoft Windows?
+
+![new user retention windows](../images/blog/retention-vs-churn/new-user-retention-windows-users.png)
+
+It's obvious Windows users retain worse than the baseline, dipping as low as 15% in week 4 onwards in the 'Dec 11' cohort. 
+
+Do macOS users retain better than Windows users? Let's check...
+
+![new user retention mac os](../images/blog/retention-vs-churn/retention-table-macos.png)
+
+They do! While the exact values vary, macOS users consistently retain better than Windows users. 
+
+This prompts several avenues os investigation. Is the Windows Hedgebox experience worse and, if so why? Does Hedgebox solve some fundamental problem for macOS users that makes them retain better? Are Windows users demographically different to macOS users?
+
+This is just one example using demo data in PostHog. You could also filter users by:
+
+- Their age
+- The browser they use
+- The email account they used to sign up 
+- The country or city they live in
+- The number of files they upload
+- The types of files they upload
+- User job titles, company type or myriad other properties
+
+Cohort retention tables are also useful for tracking revenue retention, especially if you charge based on usage. Positive revenue retention (i.e. more than 100%) indicates users increase their spend over time. Needless to say, this is a good thing.
+
+> 😱 **Pedantic Sidebar:** For those thinking "if retention and churn rate are just the inverse of each other, why can't I create a churn rate table instead?" My answer is this... yes, you could, but it would be a dumb thing to do. Why? Because retention tables make more intuitive sense. Be a rebel if you like, but you're dead to me. 
```

Comment:
I don't get this argument. It feels like another 'because I said so.' Why would it be dumb? You've just explained what a retention table is to me, so I (the casual reader) have no idea what a churn table looks like.